qiskit_experiments/randomized_benchmarking/rb_experiment.py

```diff
@@ -35,14 +35,12 @@ class StandardRB(BaseExperiment):
     """Standard Randomized Benchmarking Experiment class.
 
     Overview
-        Randomized Benchmarking (RB) is an efficient and robust method
-        for estimating the average error-rate of a set of quantum gate operations.
 
```

Comment:
Delete the new line so that `Overview` will be understood as a section and will look like this: 
![image](https://user-images.githubusercontent.com/46826214/126075517-7691f702-d87e-420c-9031-e8f3986ba741.png)

and not like this: 
![image](https://user-images.githubusercontent.com/46826214/126075540-7cf6e617-e38f-4a8f-ad94-23692be3c899.png)